CHANGELOG.md

```diff
@@ -75,6 +75,24 @@ If you see unexpected behaviour, make sure the `id` for the  textarea is unique
 
 This change was introduced in [pull request #2408: Prevent issues with character count when textarea `id` includes CSS syntax characters](https://github.com/alphagov/govuk-frontend/pull/2408).
 
+#### Include JavaScript for the skip link to improve screen reader announcements
+
+We've added JavaScript for the skip link component to set focus to the target element. This improves the announcements especially for Mac VoiceOver users and also for JAWS users.
```

Comment:
Should we give an example of what we mean by 'target element' (if there could be any inexperienced users who wouldn't understand this right away)? "The main content on a page"?

How does it improve the announcements? By reading the main element when users activate the skip link - which is different to what happened before we made this change?

Last comment: we had some feedback from release notes testing that some of our users are junior devs who are inexperienced when it comes to accessibility. Do we need to spell out the JAWS acronym for them, or provide any links?

```suggestion
We've added JavaScript for the skip link component to set focus to the target element, for example, the main content on a page. This change improves the announcements, especially for users of Mac VoiceOver and JAWS.
```

Reply:
Thanks for the feedback Eoin, all really good suggestions. I've gone with something that's quite close to what you suggested including removing some of the jargon. Have a look and let me know if you think it needs further tweaking.